sympy/printing/latex.py

```diff
@@ -2452,6 +2477,12 @@ def latex(expr, fold_frac_powers=False, fold_func_brackets=False,
     gothic_re_im : boolean, optional
         If set to ``True``, `\Re` and `\Im` is used for ``re`` and ``im``, respectively.
         The default is ``False`` leading to `\operatorname{re}` and `\operatorname{im}`.
+    decimal_separator : string, optional
+        Specifies what separator to use to separate the whole and fractional parts of a
+        floating point number as in `2.5` (for the default, "period") or `2{,}5`
+        when "comma" is specified. Lists, sets, and tuple are printed with semicolon
+        separating the elements when 'comma' is chosen([1; 2; 3] when comma is chosen and
```

Comment:
Space before `(` and `"comma"`

Reply:
Let me change the last comment to double backticks.

Reply:
I've added a space before ( on line 2484 and changed all the ' to " around the words period and comma! Also, I've been using quotes instead of backticks.

Reply:
Compare to how the other options are described. They use double backticks.

Reply:
Sorry, didn't notice. Updated.